fms_core/fzy.py

```diff
@@ -0,0 +1,207 @@
+
```

Comment:
We should paste in the MIT license from the original library, and maybe mention in readme that portions arent (c) us